docs/sql/statements/create-subscription.rst

```diff
@@ -102,6 +102,12 @@ Parameters
 
   ``password``: user password.
 
+  .. CAUTION::
+
+     The user specified in the connection string, must have ``DQL`` privileges 
+     on all tables of the publication on a publisher cluster. Tables, for which
+     the user don't have the ``DQL``  privilege, will not be replicated.
```

Comment:
```suggestion
     the user does not have the ``DQL``  privilege, will not be replicated.
```

Probably should be does not because of 3rd person singular user